CHANGES.txt

```diff
@@ -23,6 +23,7 @@ To be included in 1.0.0 (unreleased)
 * Fix error packet handling for SSCursor #428
 * Required python version is now properly documented in python_requires instead of failing on setup.py execution #731
 * Add rsa extras_require depending on PyMySQL[rsa] #557
+* Self-reported __version__ now returns version from installed package metadata, otherwise unknown #734
```

Comment:
How about
```suggestion
* Self-reported `__version__` now returns version from installed package metadata, otherwise `'unknown'` #734
```

Reply:
I want to reformat the changelog before the next release anyways to implement proper formatting and wording.